user-access-event-handlers/src/test/java/no/unit/nva/useraccess/events/EventBasedScanHandlerTest.java

```diff
@@ -203,7 +205,7 @@ private ScanDatabaseRequest createSampleScanRequest(Integer pageSize) {
         return new ScanDatabaseRequest(IDENTITY_SERVICE_BATCH_SCAN_EVENT_TOPIC, pageSize, NO_START_MARKER);
     }
 
-    private static class FamilyNameChangeMigration implements MigrationService {
+    private static class FamilyNameChangeMigration implements UserMigrationService {
```

Comment:
This is a test class inside the Handler's unit tests. It performs a stupid migration to the users in order to verify that the users have indeed been migrated.